### PR TITLE
Upgrade to ember cli 2.15

### PR DESCRIPTION
**Pull request by cherewaty**
Gets rid of bower 🎉

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,25 +20,19 @@ module.exports = {
...
 
-    // take the transpiled x-request sources and put them into
-    // `modules/x-request/{*}.js` so that the
-    // ember-cli build will pick them up.
-    var xRequest = new Funnel(transpiled, {
-      destDir: 'modules/x-request'
-    });
+    // transpile the x-request sources into whatever the consuming app wants
+    var babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel');
+    var transpiled = babelAddon.transpileTree(xRequest);
```
Comment (cherewaty) [inline index.js:33]: This was the hardest part.

File: config/ember-try.js [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+        devDependencies: {
+          'ember-source': null
+        }
+      }
+    },
+    {
+      name: 'ember-lts-2.12',
+      npm: {
+        devDependencies: {
+          'ember-source': '~2.12.0'
```
Comment (Robdel12) [inline config/ember-try.js:25]: Curious, what's the difference between `components/ember#lts-2-8` and just specifying the version number?

Comment (cherewaty) [reply config/ember-try.js:25]: I think the `lts` tag isn't necessarily applied to `2.8.0` - the `lts` will pick up whichever 2.8.x release got the `lts` designation. (I just went with `ember-cli`'s default `ember-try` blueprint here.)

Comment (Robdel12) [reply config/ember-try.js:25]: Sounds good to me! :D

**Review by Robdel12 — APPROVED** [review]

**Pull request merged**